content/concepts/products.md

```diff
@@ -0,0 +1,746 @@
+# Products
+
+The Product is the central resource of our PIM, seems legit since we are doing Product Management. :wink:
```

Comment:
```suggestion
The Product is the central resource of our PIM, it makes sense right since we are doing Product Management. :wink:
```

Reply:
Je trouve legit très informel

Reply:
```suggestion
The **Product** is the central resource of our PIM and, when you think about it, it makes perfect sense since what we are doing is Product Management. :wink:
```